cmd/ocsp-responder/main.go

```diff
@@ -50,6 +50,25 @@ type Config struct {
 		// upstream's timeout when making request to ocsp-responder.
 		Timeout cmd.ConfigDuration
 
+		// The worst-case freshness of a response during normal operations.
+		//
+		// This controls behavior when both Redis and MariaDB backends are
+		// configured. If a MariaDB response is older than this, ocsp-responder
+		// will try to serve a fresher response from Redis, waiting for a Redis
+		// response if necessary.
+		//
+		// This is related to OCSPMinTimeToExpiry in ocsp-updater's config,
+		// and both are related to the mandated refresh times in the BRs and
+		// root programs (minus a safety margin).
+		//
+		// This should be configured slightly higher than ocsp-updater's
+		// OCSPMinTimeToExpiry, to account for the time taken to sign
+		// responses once they pass that threshold. For instance, a good value
+		// would be: OCSPMinTimeToExpiry + OldOCSPWindow.
```

Comment:
```suggestion
		// This should be configured slightly higher than ocsp-updater's
		// ExpectedFreshness, to account for the time taken to sign
		// responses once they pass that threshold. For instance, a good value
		// would be: ExpectedFreshness + OldOCSPWindow.
```

Reply:
Kept as-is, since I reverted the rename of OCSPMinTimeToExpiry.